src/client/pages/forum/Forum.jsx

```diff
@@ -1,13 +1,26 @@
 import Post from "./Post";
+import { useState, useEffect } from "react";
 
 
 const Forum = () => {
+    const [posts, setPosts] = useState([]);
+
+    useEffect(() => {
+        const fetchPosts = async () => {
+            const res = await fetch('http://localhost:4000/post')
```

Comment:
For consistency purposes, I think the URL should be added to the config.js

Reply:
This is @hannah68's domain. She has been informed of the above, not sure why this PR includes these changes.

Reply:
I don't have config file yet, I'll fix that.